modules/apps/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/internal/upgrade/v4_0_1/DDMFormInstanceRecordUpgradeClassNames.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.workflow.kaleo.internal.upgrade.v4_0_1;
+
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.LoggingTimer;
+import com.liferay.portal.upgrade.util.Table;
+import com.liferay.portal.workflow.kaleo.internal.upgrade.v1_3_0.BaseUpgradeClassNames;
+import com.liferay.portal.workflow.kaleo.runtime.util.WorkflowContextUtil;
+
+import java.io.Serializable;
+
+import java.util.Map;
+
+/**
+ * @author Carolina Barbosa
+ */
+public class DDMFormInstanceRecordUpgradeClassNames
```

Comment:
I saw the classes that extend from BaseUpgradeClassNames, and all start with **Upgrade**, and we can use ClassName in the singular.
Could you rename it to UpgradeDDLFormRecordClassName ?

Should we name the class focusing on the old value? It was the old value that got upgraded. What do you think?

Reply:
From what I know, the naming pattern for classes that extend a Base class is:

`different prefix` + `base class suffix` 

like https://github.com/brianchandotcom/liferay-portal/pull/152933#issuecomment-2269496320

But now I'm confused

Reply:
See also https://github.com/liferay/liferay-portal/commit/0db24203e67df4ee2d1a87c6312dec1182778ac8#diff-c75cabcfac00850e6174e0849471e8dc1fb1c11690e2a4f1460bbc4716e89e35R23